How to run multiple instances with docker

Tako88 opened the pull request:
Basic guide on how to run multiple instances using docker.

## Summary

To add information about running multiple instances of freqtrade with docker.

Tako88 committed:
How to run multiple instances with docker

Basic guide on how to run multiple instances using docker.

xmatthias commented on docs/advanced-setup.md:
```diff
@@ -52,6 +52,71 @@ freqtrade trade -c MyConfigUSDT.json -s MyCustomStrategy --db-url sqlite:///user
+    # Please read the https://www.freqtrade.io/en/latest/rest-api/ documentation
+    # before enabling this.
+    ports:
+      - "127.0.0.1:8081:8081"
```
I'd think that a better way (especially when using modular configs) is to use the following port mapping.
This way, the bot still litens on 0.0.0.0:8080, but you connect via 127.0.0.1:8081.


```suggestion
      - "127.0.0.1:8081:8080"
```

To me, this is easier, but it requires pretty good understanding of this concept - so i'd leave this up to you...

Tako88 replied:
Oh good catch, agreed.

xmatthias commented on docs/advanced-setup.md:
````diff
@@ -52,6 +52,71 @@ freqtrade trade -c MyConfigUSDT.json -s MyCustomStrategy --db-url sqlite:///user
+### Multiple instances using docker
+
+To run multiple instances of freqtrade using docker you will need to edit the docker-compose.yml file and add all the instances you want as separate services. Remember, you can separate your configuration into multiple files, so it's a good idea to think about making them modular, then if you need to edit something common to all bots, you can do that in a single config file. 
+```
````
```suggestion
``` yml
```

xmatthias commented on docs/advanced-setup.md:
````diff
@@ -52,6 +52,71 @@ freqtrade trade -c MyConfigUSDT.json -s MyCustomStrategy --db-url sqlite:///user
+
+```
+You can use whatever naming convention you want, freqtrade1 and 2 are arbitrary.
+
````
Please add a note here which states that the above restrictions (different databases, different telegram bots) still applies

Tako88 committed:
Update docs/advanced-setup.md

Co-authored-by: Matthias <xmatthias@outlook.com>

Tako88 committed:
Update docs/advanced-setup.md

Co-authored-by: Matthias <xmatthias@outlook.com>

Tako88 committed:
Update advanced-setup.md

A note on having to use different database files, ports and telegram configs for each bot.

xmatthias reviewed (approved):
LGTM :+1:
